src/pages/ResearcherProfile.js

```diff
@@ -1,4 +1,4 @@
-import _, { isNil, isEmpty } from 'lodash';
+import {omit, cloneDeep, isEmpty, isNil, get, trim, omitBy } from 'lodash';
```

Comment:
There's a second lodash import statement on line 13 where it's being aliased as ```ld```, so the entire library is still being imported.

The ```ld``` alias is being used on lines 463 and 572

Reply:
Totally missed that ... will clean that up too.